convert2rhel/backup/files.py

```diff
@@ -118,15 +118,9 @@ def restore(self, rollback=True):
             loggerinst.info("%s hasn't been backed up." % self.filepath)
             return
 
-        try:
-            shutil.copy2(self._backup_path, self.filepath)
-            os.remove(self._backup_path)
-        except (OSError, IOError) as err:
-            # Do not call 'critical' which would halt the program. We are in
-            # a rollback phase now and we want to rollback as much as possible.
-            # IOError for py2 and OSError for py3
-            loggerinst.critical_no_exit("Error(%s): %s" % (err.errno, err.strerror))
-            return
+        # Possible exceptions will be handled in the BackupController
+        shutil.copy2(self._backup_path, self.filepath)
+        os.remove(self._backup_path)
```

Comment:
For good measure lets add this to the docstring. That this function throws OSError, IOError. Ask in team chat for syntax if you need it. @r0x0d probably knows if you can't find it

Reply:
It should be: 

```python
"""
<title>

<description>
:raises OSError: why it raises oserror
:raises IOError: Usually, same as the above ^ 
"""
```

Reply:
Thinking more about this now, maybe we should even consider adding a `warning` in the docstring to mention that whatever exception raised here will bubble up and be handled by the caller of this plugin. 

```python
"""
<title>

<description>

     .. warning::
         Add the warning message here

:raises OSError: why it raises oserror
:raises IOError: Usually, same as the above ^ 
"""
```

Reply:
Agreed, sounds good!

Reply:
Thanks! I'll add those comment improvements